<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_main">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="os_info">
      <attribute name="title">
       <string>系统信息</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string>CPU信息</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>CPU名字</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>CPU核数</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>CPU频率</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>491</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="cpu_name">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>441</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="cpu_core">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>431</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="cpu_frequency">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string>版本信息</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>内核版本</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>编译器版本</string>
       </property>
      </widget>
      <widget class="QLabel" name="kernel_version">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>371</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="gcc_version">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>200</y>
         <width>371</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>220</y>
         <width>491</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>240</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string>详细信息</string>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>259</y>
         <width>471</width>
         <height>161</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>469</width>
          <height>159</height>
         </rect>
        </property>
        <widget class="QTextEdit" name="textEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-1</y>
           <width>471</width>
           <height>161</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>100</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>缓存大小</string>
       </property>
      </widget>
      <widget class="QLabel" name="cache_size">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>100</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>缓存大小</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="process_management">
      <attribute name="title">
       <string>资源管理</string>
      </attribute>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;CPU使用率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;内存使用率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>90</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>411</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>190</y>
         <width>411</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>220</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;swap使用率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>220</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>260</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>总大小</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="swap_total">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>已使用 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="swap_used">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>未使用</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="swap_free">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>总内存</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mem_total">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>已使用 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mem_used">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>未使用</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mem_free">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>160</y>
         <width>226</width>
         <height>16</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Cached</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cached">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Buffers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="buffer">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="resource_management">
      <attribute name="title">
       <string>进程管理</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>1</y>
         <width>631</width>
         <height>391</height>
        </rect>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>400</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>进程总数</string>
       </property>
      </widget>
      <widget class="QLabel" name="process_number">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>400</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>111</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>400</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string> 运行中进程数目</string>
       </property>
      </widget>
      <widget class="QLabel" name="process_number_running">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>400</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancel_process_btn">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>400</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>结束该进程</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="about_me">
      <attribute name="title">
       <string>关于我</string>
      </attribute>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>11</y>
         <width>601</width>
         <height>411</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>90</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>QQ: 1207680294</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
